docs/fastdds/getting_started/simple_app/includes/task_publisher.rst

```diff
@@ -0,0 +1,157 @@
+Write the Fast DDS publisher
+""""""""""""""""""""""""""""
+
+From the `src` directory in the workspace, run the following command to download the HelloWorldPublisher.cpp file.
+
+.. code-block:: bash
+
+    wget -O HelloWorldPublisher.cpp \
+        https://raw.githubusercontent.com/eProsima/Fast-RTPS-docs/master/code/Examples/C++/DDSHelloWorld/src/HelloWorldPublisher.cpp
+
+Now you have the publisher's source code. The publisher is going to send 10 publications under the topic HelloWorld.
+
+.. literalinclude:: /../code/Examples/C++/DDSHelloWorld/src/HelloWorldPublisher.cpp
+  :language: C++
+  :linenos:
+
+Examine the code
+****************
+
+At the beginning of the file we have a Doxygen style comment block with the ``@file`` field that tells us the name of
+the file.
+
+.. literalinclude:: /../code/Examples/C++/DDSHelloWorld/src/HelloWorldPublisher.cpp
+    :language: C++
+    :lines: 15-18
+
+Below are the includes of the C++ headers.
+The first one includes the HelloWorldPubSubTypes.h file with the serialization and deserialization functions of the
+data type that we have defined in the previous section.
+
+.. literalinclude:: /../code/Examples/C++/DDSHelloWorld/src/HelloWorldPublisher.cpp
+    :language: C++
+    :lines: 20
+
+The next block includes the C++ header files that allow the use of the Fast DDS API.
+
+*   DomainParticipantFactory. Allow the creation and destruction of DomainParticipant objects.
+*   DomainParticipant. Acts as a container for all other Entity objects and as a factory for the Publisher, Subscriber,
```

Comment:
In general, each time you name a class from the API, you could add a reference to the API-ref (when it gets merged). It's just so we do not forget to do that